test/newrelic.test.js

```diff
@@ -15,7 +15,7 @@ governing permissions and limitations under the License.
 "use strict";
 
 const NewRelic = require('../lib/newrelic');
-const MetricsTestHelper = require('../lib/testhelper');
+const MetricsTestHelper = require('./testhelper');
```

Comment:
Do not move this file. It is exported as API and has to be in `lib/` for that.

Reply:
It's not exported in index.js, so it's an api you have to pick directly and go look into the folder?
(Or is the missing index.js export an oversight?)

Reply:
Ahh, sorry, that happens in commons, which would break with this change: https://github.com/adobe/asset-compute-commons/blob/master/index.js#L23

So just leave it in `libs/`.

Do we know this has all the recent changes from the internal repo that got lost earlier?

/cc @DrGirlfriend

Reply:
I'm going to sync after this PR tests green and is approved, @alexkli

Reply:
They'll be here: https://github.com/adobe/node-openwhisk-newrelic/pull/6